nss-day-cohort-48/rare-api-boastful-bulldogs
# User can edit a comment

User can now edit a comment

## Steps to test
(pull client too)
1. git fetch --all
2. checkout to anm-api-editcomment
3. go to a post's comments and you should only be able to edit your own comments
4. edit comment
5. also verify there are edit and delete buttons only for your comments
